guide/additional-features/cooldowns.md

```diff
@@ -0,0 +1,94 @@
+# Cooldowns
+
+Spam is something you generally want to avoid–especially if one of your commands requires calls to other APIs or takes a bit of time to build/send. Cooldowns are also a very common feature bot developers want to integrate into their projects, so let's get started on that!
+
+This section assumes you followed the [Command Handling](/guide/creating-your-bot/command-handling.md) part.
+
+First, add a cooldown key to one of your commands (we use the ping command here) exports. This determines how long the user will have to wait (in seconds) before using this specific command again.
+
+```js {3}
+const { SlashCommandBuilder } = require('discord.js');
+
+module.exports = {
+	cooldown: 5,
+	data: new SlashCommandBuilder()
+		.setName('ping')
+		.setDescription('Replies with Pong!'),
+	async execute(interaction) {
+		// ...
+	},
+};
+```
+
+In your main file, initialize an empty [Collection](/guide/additional-info//collections.md) which you can then fill later when commands are used:
+
+```js {2}
+client.commands = new Collection();
+client.cooldowns = new Collection();
+```
+
+The keys will be the command names, and the values will be Collections associating the user id (key) to the last time (value) this specific user used this specific command. Overall the logical path to get a specific user's last usage of a specific command will be `cooldowns > command > user > timestamp`.
+
+In your main file, add the following code:
+
+```js {1,3-5,7-9,11-13}
+const { cooldowns } = client;
+
+if (!cooldowns.has(command.data.name)) {
+	cooldowns.set(command.data.name, new Collection());
+}
+
+const now = Date.now();
+const timestamps = cooldowns.get(command.data.name);
+const cooldownAmount = (command.cooldown || 3) * 1000;
```

Comment:
maybe defaultDuration or smth like that for that `3` 
as now it looks like random number